pxc/jenkins/pxc-package-testing-test-yash.groovy

```diff
@@ -0,0 +1,310 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/panchal-yash/jenkins-pipelines.git'
+]) _
+
+void installDependencies() {
+    sh '''
+        export PATH=${PATH}:~/.local/bin
+        sudo yum install -y git python3-pip jq
+        sudo amazon-linux-extras install ansible2
+        python3 -m venv venv
+        source venv/bin/activate
+        python3 -m pip install setuptools wheel
+        python3 -m pip install molecule==2.22 boto boto3 paramiko
+    '''
+    
+    sh '''
+        rm -rf package-testing
+        git clone https://github.com/panchal-yash/package-testing --branch wip-pxc-package-testing
+    '''
+
+    sh '''
+
+        echo "Private IP"
+        curl http://169.254.169.254/latest/meta-data/local-ipv4
```

Comment:
What is this IP?

Reply:
Hi they are internal AWS IPs for fetching instance metadata. I was using them for confirming and debugging.

Will remove them